packages/webapp/package.json

```diff
@@ -22,10 +22,12 @@
         "@testing-library/dom": "8.20.1",
         "@testing-library/jest-dom": "6.2.0",
         "@testing-library/react": "12.1.5",
+        "@types/lodash": "4.17.7",
         "@types/react": "17.0.62",
         "@types/react-copy-to-clipboard": "5.0.7",
         "@types/react-dom": "17.0.20",
         "@types/react-redux": "7.1.33",
+        "@types/redux-logger": "3.0.9",
```

Comment:
Same here, we should use `3.0.13` here.
This is the latest version and also has the dist-tag `ts4.9` which corresponds to the typescript version that we have used https://www.npmjs.com/package/@types/redux-logger?activeTab=versions

Reply:
Done

Reply:
@hitesh-parmar Actually we will keep that version as it's not compatible with redux v4